.styleci.yml

```diff
@@ -1,8 +1,20 @@
 preset: psr2
 
 enabled:
+  - concat_with_spaces
```

Comment:
Sorry, I would say `without` in https://github.com/greg0ire/enum/pull/43#issuecomment-227983772.